Makefile

```diff
@@ -18,8 +18,8 @@ GO_LDFLAGS += -s -w
 
 ARTIFACTS := _out
 
-TOOLS ?= ghcr.io/siderolabs/tools:v1.3.0-1-g712379c
-PKGS ?= v1.3.0
+TOOLS ?= ghcr.io/siderolabs/tools:v1.4.0
```

Comment:
we have that updated with new Go 1.20, better to take values from 1.4.0 of Talos `Makefile`

Reply:
ok, used the same from `v1.4.0` tag of Talos.